var/exp_repo/experiments/scalingexperiment/experiment.py

```diff
@@ -0,0 +1,124 @@
+from benchpark.directives import variant
+
+
+class ScalingExperiment(object):
+    variant(
+        "scaling-factor",
+        default="2",
+        values=int,
+        description="Factor by which to scale values of problem variables",
+    )
+
+    variant(
+        "scaling-iterations",
+        default="4",
+        values=int,
+        description="Number of experiments to be generated",
+    )
```

Comment:
#379 was merged, but these are not conditioned on `when=` (as mentioned in https://github.com/LLNL/benchpark/pull/375#discussion_r1775946142). I think you can effectively turn scaling off by setting `scaling-iterations=1`, although it still might be clearer to use `when`.